README.md

```diff
@@ -118,21 +122,36 @@ This deep link routing callback is called 100% of the time on init, with your li
             // route to a profile page in the app for Joe
             // show a customer welcome
         }
-	}];
+    }];
 }
 ```
 
 ```objc
 - (BOOL)application:(UIApplication *)application openURL:(NSURL *)url sourceApplication:(NSString *)sourceApplication annotation:(id)annotation {
-	// pass the url to the handle deep link call
-	// if handleDeepLink returns YES, and you registered a callback in initSessionAndRegisterDeepLinkHandler, the callback will be called with the data associated with the deep link
-	if (![[Branch getInstance] handleDeepLink:url]) {
-		// do other deep link routing for the Facebook SDK, Pinterest SDK, etc
-	}
+    // pass the url to the handle deep link call
+    // if handleDeepLink returns YES, and you registered a callback in initSessionAndRegisterDeepLinkHandler, the callback will be called with the data associated with the deep link
+    if (![[Branch getInstance] handleDeepLink:url]) {
+        // do other deep link routing for the Facebook SDK, Pinterest SDK, etc
+    }
     return YES;
 }
 ```
 
+If you want to use your test app during development, in application:didFinishLaunchingWithOptions: you can get the Branch object like this:
+
+```objc
+Branch *branch = [Branch getTestInstance];
+```
+
+Or
+
+```objc
+Branch *branch = [Branch getInstance:@"your test branch key"];  // replace with your actual branch key
+```
+
+Either way, we recommend you put a #warning directive to remind you to change back to live app during deployment later.
+Also, note the Branch object is singleton, so you can and should still use `[Branch getInstance]` in all the other places (see examples below).
```

Comment:
Hmm, this seems confusing to me. Calling `getTestInstance` once will cause all future calls to `getInstance` to return the test instance... feels a little strange.